rules/do-you-use-strongly-typed-ids/rule.md

```diff
@@ -129,7 +129,9 @@ Dan suggests to use `struct` or `class` instead because generating 10,000 strong
 
 However, `class` and `struct` are more verbose and require that we implement the `Equals` and `GetHashCode` overrides manually!
 
-Dan unfortunately missed out a benchmark for `record struct` which turns out to give us the best of both worlds - performance and succinct one liner declarations without manually implementing equality checks.
+Dan unfortunately missed out a benchmark for `record struct`. We wrote our own benchmark's similar to Dan's and as it turns out `record struct` gives us the best of both worlds! We get the performance of `struct` and the succinct one liner declarations of `record` without needing to manually implement the equality checks.
+
+We can also add the `readonly` modifier to indicate that instances our strongly typed IDs are immutable - the compiler will warn us at build time when any unwanted mutation to the ID is made.
```

Comment:
Aren't records already immutable though?

Reply:
correct, `records` are immutable by default :)

`record structs` are not :)...

don't you just love C#? :)

adding `readonly` to `record struct` makes it immutable again.

Here is a nice SO example: https://stackoverflow.com/q/73026919